modules/packages/zypper.in

```diff
@@ -103,9 +103,18 @@ def subprocess_Popen(cmd, stdout=None, stderr=None):
     return result
 
 
+# If subprocess.PIPE is passed as stderr, it will re-write it in an ErrorMessage
 def subprocess_call(cmd, stdout=None, stderr=None):
     process = subprocess_Popen(cmd, stdout, stderr)
-    return process.wait()
+    return_code = process.wait()
```

Comment:
This should use `process.communicate()` because otherwise the process may be stuck writing to its stderr while nothing is reading from it creating a deadlock here.

Reply:
Fixed in a new commit, I will rebase if needed.